src/app/tests/suites/certification/Test_TC_S_2_6.yaml

```diff
@@ -52,114 +105,243 @@ tests:
       PICS: S.S.C03.Rsp
       verification: |
           ./chip-tool scenesmanagement remove-all-scenes 0x0000 2 1 --commissioner-name beta
+
+          Verify the RemoveAllScenesResponse with following fields:
+          Status is SUCCESS
+          Group ID is 0x0000  on the TH(Chip-tool) log and below is the sample log provided for the raspi platform:
+
+          [1700826575.191275][15971:15973] CHIP:DMG: Received Command Response Data, Endpoint=1 Cluster=0x0000_0062 Command=0x0000_0003
+          [1700826575.191321][15971:15973] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0062 Command 0x0000_0003
+          [1700826575.191938][15971:15973] CHIP:TOO:   RemoveAllScenesResponse: {
+          [1700826575.191974][15971:15973] CHIP:TOO:     status: 0
+          [1700826575.191985][15971:15973] CHIP:TOO:     groupID: 0
+          [1700826575.191995][15971:15973] CHIP:TOO:    }
+
           ./chip-tool scenesmanagement remove-all-scenes 0x0000 3 1 --commissioner-name gamma
+
+          Verify the RemoveAllScenesResponse with following fields:
+          Status is SUCCESS
+          Group ID is 0x0000  on the TH(Chip-tool) log and below is the sample log provided for the raspi platform:
+
+          [1700826595.190538][15975:15977] CHIP:DMG: Received Command Response Data, Endpoint=1 Cluster=0x0000_0062 Command=0x0000_0003
+          [1700826595.190578][15975:15977] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0062 Command 0x0000_0003
+          [1700826595.190637][15975:15977] CHIP:TOO:   RemoveAllScenesResponse: {
+          [1700826595.190652][15975:15977] CHIP:TOO:     status: 0
+          [1700826595.190661][15975:15977] CHIP:TOO:     groupID: 0
+          [1700826595.190669][15975:15977] CHIP:TOO:    }
       disabled: true
 
     - label: "Step 2a: TH1 reads from the DUT the SceneTableSize attribute"
       verification: |
           ./chip-tool scenesmanagement read scene-table-size 1 1
 
-          CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0005 Attribute 0x0000_0006 DataVersion: 2793536935
-          CHIP:TOO:   SceneTableSize: 16
+          Verify the "SceneTableSize" attribute value is 16 which is recorded into SceneTableSize on the TH (Chip-tool) and below is the sample log provided for the raspi platform:
```

Comment:
Is this value always 16? or for some DUTs can this be a different value? is 16 the minimum?

Reply:
16 is the minimum